README.txt
~~~~~~~
Version
~~~~~~~

This is BETA version 0.8.5.0 - See warning below!

~~~~~~~~~~~~~~~~~~~~~~
WARNING - BETA VERSION
~~~~~~~~~~~~~~~~~~~~~~

This software is BETA, that means it should be used for TESTING ONLY.
Do NOT install this software on your production system or use it to store
important information.  Unless you are testing this software, you should be
using the latest stable release (0.8.0.3).

~~~~~~~~~~~~~~
Reporting Bugs
~~~~~~~~~~~~~~

It is absolutely critical for you to report any bugs you find with this software.
If you don't, they can not be fixed.  There is no point to installing and testing
this BETA relase software if you do not provide your feedback.  If you find a bug
please thoroughly search the tracker to make sure it's not already known.  If you
are certain you have discovered a NEW bug, please log it into the bug tracker located
at http://www.projectpier.org/project/issues/ under version 0.8.5.x-dev.

If you have tested this beta version and have NOT found any bugs, we need to hear that 
also!  Please let us know what type of system you are using and the extent of your 
testing.  Please include the OS, Apache, PHP and MySQL versions and/or the name of
the web hosting provider the testing was performed on.  A new forum has been setup
specifically to gather and discuss this format, it is located at 
http://www.projectpier.org/forum/development/beta

~~~~~~~~~~~~~~~~~~~
System requirements
~~~~~~~~~~~~~~~~~~~
	
ProjectPier requires a web server, PHP (5.0.2 or greater with MySQL, GD, and 
SimpleXML extensions) and MySQL with InnoDB support. The recommended web 
server is Apache, but IIS 5 and above have been reported to work also.
	
ProjectPier is not PHP4 compatible and it will not run on PHP versions prior
to PHP 5.0.2.
	
Recommended configuration:
	
: PHP 5.1 or greater
: MySQL 4.1 or greater with InnoDB support (see notes below)
: Apache 2.0 or greater
        
If you do not have these installed on a server or your personal computer, 
you can visit the sites below to learn more about how to download and install
them.  They are all licensed under various compatible Open Source licenses.	
: PHP    : http://www.php.net/
: MySQL  : http://www.mysql.com/
: Apache : http://www.apache.org/
	
~~~~~~~~~
Upgrading
~~~~~~~~~

If you are upgrading from ProjectPier 0.8.0.3, do not follow the installation 
steps below, instead see upgrade.txt for an upgrade procedure.  You should NOT
be upgrading your production installation, this version is for testing only!

~~~~~~~~~~~~
Installation
~~~~~~~~~~~~
	
1. Make sure your server meets the requirements, see notes below regarding
   enabling InnoDB support.
2. Download ProjectPier from the project website - http://www.projectpier.org/.
3. Unpack and upload the files to your web server.
4. Direct your browser to the /public/install directory and follow the installation
   procedure.

~~~~~~~~~~~~~~~~~~~~~~~
Enabling InnoDB Support
~~~~~~~~~~~~~~~~~~~~~~~

Some installations of MySQL don't support InnoDB by default.  The ProjectPier installer 
will tell you if your server is not configured to support InnoDB. This is easy to fix: 

1. Open your MySQL options file, the file name is my.cnf (Linux) - usually at /etc/my.cnf 
   or my.ini (Windows) - usually at c:/windows/my.ini.  If you are using the Uniform Server
   on Windows, the file will be named 'my-small' and will need to be edited with a unix 
   compatible editor such as PSPad or EditPad Lite.
2. Comment the skip-innodb line by adding # in front of it (like #skip-innodb). 
3. It would also be good to increase max_allowed_packet to ensure that 
   you'll be able to upload files larger than 1MB. Just add this 
   line bellow #skip-innodb line: 
   set-variable = max_allowed_packet=64M

~~~~~~~~~~~~~~~~~~~~~
Changing the Language
~~~~~~~~~~~~~~~~~~~~~

ProjectPier installation screens are in English and English is the default language
for the program.  However, ProjectPier is distributed with 22 other language options.

After installation is complete, the language can be changed by manually editing the
file /config/config.php. 

How: Find this line in your config.php file: define('DEFAULT_LOCALIZATION', 'en_us');
     Change en_us to the appropriate language abbreviation (select from the list below).
     Example for German: define('DEFAULT_LOCALIZATION', 'de_de');

The following languages are available:
es_ar = Argentinian Spanish
pt_br = Bralilian Portuguese
bg_bg = Bulgarian
pt_pt = Continental Portuguese
cz_cz = Czech
da_dk = Danish
nl_nl = Dutch
en_us = English (US)
fi_fi = Finnish
fr_fr = French
de_de = German
el_gr = Greek
hu_formal = Hungarian (formal)
hu_informal = Hungarian (informal)
it_it = Italian
lt_lt = Lithuanian
es_mx = Mexican Spanish
pl_pl = Polish
ru_ru = Russian
zn_ch = Simplified Chinese
sl_si = Slovenian
sv_sv = Swedish
tr_tr = Turkish

~~~~~~~~~~~~~~~~~
About ProjectPier
~~~~~~~~~~~~~~~~~
	
ProjectPier is an Open Source project management and collaboration
tool that you can install on your own server. It is released under the 
terms of the Gnu Affero General Public License (AGPL) (see LICENSE.txt for details).  
	
: http://www.projectpier.org


CHANGES.txt
           		
~~~~~~~~~~~~~~~~~~~~~~
ProjectPier Change Log
~~~~~~~~~~~~~~~~~~~~~~

--------------------------------
Version 0.8.5.0 Beta 1 June 2009
--------------------------------

Features:
- Added (Node 28) - A calendar view of milestones
- Added two new themes: "zura1" and "zura-blue-steel"
- Added 8 new translations: Bulgarian, Czech, Mexican Spanish,
  Finnish, French, Italian, Lithuanian, Continental Portuguese,
  and Russian

Bug Fixes:

- Fixed (Node 685) - Milestones coming due the next day said "one days"
  instead of "one day".
- Fixed (Node 506) - Type "Create new account for you" should be "Create
  a new account for you".
- Fixed (Node 721) - No email notification when comments are added.
- Fixed (Node 843) - Cookies not expiring with browser closes
- Fixed (Node 797) - Remove redundant tags automtically
- Fixed (Node 743) - GIF format company logo with transparency now displays
  properly.
- Fixed (Node 1262) - Documentation for ProjectPier not consistent with
  the project's code standards: capatilized file names where needed.
- Fixed (Node 1038) - Pagination for search now works correctly.
- Fixed (Node 1237) - Comments to private messages are now forced to be private.
- Fixed (Node 437) - Security: Users can no longer download files from
  projects they are not assigned to by manipulating the URL.

----------------------------
Version 0.8.0.3 16 June 2008
----------------------------

- Fixed (Node 721) - Correct variable misspelling
- Fixed (Node 588) - Fixed DB scheme for storing timezone

----------------------------
Version 0.8.0.2  15 Feb 2008
----------------------------

- Fixed (Node 629) - Added missing XSRF files from 0.8.0.1
- Minor code cleanups

----------------------------
Version 0.8.0.1  12 Feb 2008
----------------------------

- Fixed (Node 665) - multiple XSS vulnerabilities
- Fixed (Node 629) - XSRF when deleting companies, users or projects
- Fixed (Node 455) - code unified to coding standards
- Changed all files to be in Unix text file format

-----------------------------------
Version 0.8.0 (final)  22 Sept 2007
-----------------------------------
- Added new Continental Portuguese translation 
- Fixed (Node 225) - Downloading files does not work with Internet Explorer 
  under https protocol

----------------------------------------------------------
Version 0.8.0 Release Candidate 1 (0.8.0RC1)  03 Sept 2007
----------------------------------------------------------
- Fixed (Node 96) - cache, upload, and public/files folders did not exist in 
  distribution zip file
- Fixed (Node 98) - Installation - colors / line break in installation screens
  Created a whole new layout scheme for 'Complete the Installation' screen, now
  separate css from login screen.
- Fixed (Node 99) - added a link to public/install folder on 
- Fixed (Node 100) - added a 'retry' button to 2nd page on installation screen
- Fixed (Node 114) - resolved problems with mailer on some systems
- Fixed (Node 121) - A new cookie system has been implemented, each installation 
  should be completely unique now.
- Added 13 alternate languages
- Brand new default theme (old default theme included as acClassic)
- New theme 'goCollabMonochrome' included
- Message and comment email notification subject lines optimized for threading
  For Messages: ProjectTitle - MessageTitle
  For Comments: ProjectTitle - MessageTitle
  For Milestones: ProjectTitle - Milestone has been assigned to you - MilestoneName
- Modified the notification message system for Messages, Comments, and Milestones so that 
  the message body is included in the notification email (optional configuration in config.php).

----------------------------
Version 0.8-dev  23 Jun 2007
----------------------------
- Rebranded the product from activeCollab to ProjectPier
- Updated installation screens to add more information



cache/log.php
<?php die(); ?>
Session "default" started at 2010-09-28T12:49:22+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.22193002700806 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T12:49:28+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.079591035842896 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T12:49:41+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.0697021484375 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T12:53:27+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.084311962127686 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T12:53:36+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.079509019851685 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T12:59:11+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.20196890830994 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T12:59:11+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.08659291267395 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T13:01:27+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.099350929260254 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T13:01:27+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.070868968963623 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T13:01:27+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.066286087036133 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T13:01:27+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.066135883331299 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T13:01:27+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.0677490234375 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T13:01:27+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.065727949142456 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T13:01:27+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.066480875015259 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T13:01:27+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.067664861679077 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T13:01:27+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.066548109054565 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T13:03:32+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.19306588172913 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T15:01:44+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.22093105316162 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T15:55:36+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.10118412971497 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-28T15:58:55+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.11008095741272 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T08:25:51+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.0901780128479 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T08:30:31+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.089270114898682 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T08:31:01+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.093147993087769 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T08:52:46+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.11189818382263 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T08:54:38+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.088315010070801 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T08:54:44+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.090643882751465 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T08:54:57+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.098280906677246 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T08:55:00+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.088705062866211 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T09:05:25+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.08972692489624 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T09:06:49+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.089766025543213 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T09:09:55+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
#2 ERROR: Error: Undefined property:  ProjectTime::$getDoneDate in '/Users/Mac4/Sites/pier/application/views/time/index.php' on line 31 (error code: 8)
Time since start: 0.10079503059387 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T09:10:08+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.089658975601196 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T09:19:46+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.11238884925842 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T10:00:35+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.18764019012451 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T10:03:21+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.10230588912964 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T10:03:25+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.090087175369263 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T10:03:39+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.090627193450928 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T10:03:41+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.091042041778564 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T10:03:45+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.089402914047241 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T10:04:26+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.10189199447632 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T10:04:52+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.099541187286377 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T10:05:32+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.10894894599915 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T10:07:16+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.099695920944214 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T10:09:58+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.10382699966431 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T10:11:03+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.090599060058594 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T10:13:42+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.091078042984009 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T11:18:50+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.10466384887695 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T11:19:33+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.09260106086731 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T11:19:44+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.092175006866455 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T11:20:05+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.1089198589325 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T11:57:39+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.86323308944702 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T11:57:53+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.094633817672729 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T15:04:50+0000
#1 ERROR: Error: Notification failed, Template '/Users/Mac4/Sites/pier/application/views/notifier/task_assigned.php' doesn't exists
#2 ERROR: Error: Notification failed, Template '/Users/Mac4/Sites/pier/application/views/notifier/task_assigned.php' doesn't exists
#3 ERROR: Error: Notification failed, Template '/Users/Mac4/Sites/pier/application/views/notifier/task_assigned.php' doesn't exists
Time since start: 0.21908688545227 seconds
-------------------------------------------------------------------------------

Session "default" started at 2010-09-29T15:05:25+0000
#1 ERROR: Error: Undefined property:  Project::$all_times in '/Users/Mac4/Sites/pier/application/models/projects/Project.class.php' on line 535 (error code: 8)
Time since start: 0.12248396873474 seconds
-------------------------------------------------------------------------------
